libsrc/telestrat/joy_stddrv.s

```diff
@@ -0,0 +1,13 @@
+;
+; Name of the standard joystick driver
+;
+; Oliver Schmidt, 2012-11-01
+;
+; const char joy_stddrv[];
+;
+
+        .export _joy_stddrv
+
+.rodata
+
+_joy_stddrv:    .asciiz "telestrat_joy"
```

Comment:
The Makefile would convert the spelling `"telestrat/joy/telestrat-joy.s"` into `"telestrat-joy.joy"`.

I think that you should name the source file `"telestrat/joy/telestrat.s"`, so that the driver will be named `"telestrat.joy"`.

Reply:
You must change that string, too.

Reply:
I fixed the name of the driver